proto/viam/app/v1/app.proto

```diff
@@ -61,6 +61,18 @@ service AppService {
   // Resend an organization invite
   rpc ResendOrganizationInvite(ResendOrganizationInviteRequest) returns (ResendOrganizationInviteResponse);
 
+  // Third party
+
+  rpc EnableBillingService(EnableBillingServiceRequest) returns (EnableBillingServiceResponse);
+
+  rpc DisableBillingService(DisableBillingServiceRequest) returns (DisableBillingServiceResponse);
+
+  rpc UpdateBillingService(UpdateBillingServiceRequest) returns (UpdateBillingServiceResponse);
```

Comment:
do we want these in the app protos?

Reply:
I also went back and forth on this, because its related to the "org" and not billing itself, I thought it made more sense in the app proto. Open to discussion though

Reply:
i think @maxhorowitz had some thoughts on if we wanted a separate "third party service" service

Reply:
open to discussion but I think that too complex for what is 5 new RPCs right now. I think that this should follow a similar pattern to the CLI design where its a org-level thing (which lives on app).

Reply:
yeah looking again at the CLI commands-- since theyre `viam organization billing-service ... ` and not `viam third-party billing-service --org-id aoisjfdoiasjdf` (or somethign lke that) i agree where it is now makes sense